data/campaigns/Sceptre_of_Fire/scenarios/9_Caverns_of_Flame.cfg

```diff
@@ -672,15 +671,227 @@
+            [/kill]
+            [delay]
+                time=300
+            [/delay]
+            {MOVE_UNIT (id=Krawg) 22 33}
+            [store_unit]
+                [filter]
+                    id=Krawg
+                [/filter]
+                variable=krawg
+                kill=yes
+            [/store_unit]
+            {QUAKE "rumble.ogg"}
+            [scroll_to]
+                x,y=27,20
+            [/scroll_to]
+            [remove_item]
+                x,y=27,20
+            [/remove_item]
+            [terrain]
+                x=26-28,26-28,27
+                y=19,20,21
+                terrain=Qxu
+            [/terrain]
+            [kill]
+                [and]
+                    x=26-28,26-28,27
+                    y=19,20,21
+                [/and]
+                animate=no
+            [/kill]
+            [redraw]
+                side=1
+            [/redraw]
+            [delay]
+                time=300
+            [/delay]
+            [terrain]
+                [and]
+                    x=22-32
+                    y=16-24
+                    terrain=Ql
+                [/and]
+                terrain=Qlf
+            [/terrain]
+            [terrain]
+                x=26-28,26-28,27
+                y=19,20,21
+                terrain=Ql
+            [/terrain]
+            [redraw]
+                side=1
+            [/redraw]
+            [delay]
+                time=300
+            [/delay]
+            [terrain]
+                [and]
+                    terrain=Ql*
+                    radius=2
+                [/and]
+                [not]
+                    terrain=Xu*
+                [/not]
+                terrain=Qlf
+            [/terrain]
+            [redraw]
+                side=1
+            [/redraw]
+            [delay]
+                time=300
+            [/delay]
+            [color_adjust]
+                red=255
+                green=-100
+                blue=-180
+            [/color_adjust]
+            [kill]
+                x=1-44
+                animate=no
+                fire_event=no
+            [/kill]
+            {FADE_TO_BLACK}
             [message]
                 speaker=narrator
-                message= _ "Krawg puzzled over a stone tile sitting loose on the small mesa. Beneath a spattering of bat guano, there was a glowing rune. Krawg didn’t know what to make of it, so the gryphon left it for now."
+                message= _ "Once the magic seal capping the throat of the volcano was removed, lava and toxic gas exploded upward, killing everyone inside and sealing them in a tomb of hot, flowing rock. All of the gryphon riders flew up towards a hole in the roof with utmost haste, but even they could not outrun the eruption. Only Krawg managed to barely escape due to his speed and the absence of the weight of a rider on his back. The Sceptre of Fire would wait, unharmed by the lava, until some day far in the future."
```

Comment:
> Only Krawg managed to barely escape due to his speed and the absence of the weight of a rider on his back.

Not a big issue, but I feel this is unnecessarily explicit in its explanation. Maybe it's sufficient just to say Krawg was the only one fast enough to escape and let the player infer the rest.

(This string is repeated on line 1088.)